pkgs/development/python-modules/deepwave/default.nix

```diff
@@ -10,37 +10,46 @@
 , pytestCheckHook
 }:
 
+let
+  linePatch = ''
+    import os
+    os.environ['PATH'] = os.environ['PATH'] + ':${ninja}/bin'
+  '';
+in
 buildPythonPackage rec {
   pname = "deepwave";
-  version = "0.0.11";
+  version = "0.0.12";
   format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "ar4";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-d4EahmzHACHaeKoNZy63OKwWZdlHbUydrbr4fD43X8s=";
+    sha256 = "sha256-WWu0LyHlOwWMVPUy+LAszKF3VlgcqlcMlDi4oon4Dl8=";
   };
 
   # unable to find ninja although it is available, most likely because it looks for its pip version
   postPatch = ''
     substituteInPlace setup.cfg --replace "ninja" ""
+
+    # Adding ninja to the path forcibly
+    mv src/deepwave/__init__.py tmp
+    echo "${linePatch}" > src/deepwave/__init__.py
+    cat tmp >> src/deepwave/__init__.py
```

Comment:
```suggestion
    mv src/deepwave/__init__.py tmp
    echo "${linePatch}" > src/deepwave/__init__.py
    cat tmp >> src/deepwave/__init__.py
    rm tmp
```

Reply:
Thanks for the review! Indeed, it makes sense, I forgot about that. Done!